experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -2230,10 +2231,79 @@ def _clone_experiment_and_assert_common_expectations(
         return child
 
     def test_get_changelogs(self):
-        experiment = NimbusExperimentFactory.create(slug="experiment-1")
-        generate_nimbus_changelog(experiment, experiment.owner, "Test message")
-        experiment_changelogs = experiment.get_changelogs()
-        self.assertEqual(experiment_changelogs[0].message, "Test message")
+        experiment = NimbusExperimentFactory.create(
```

Comment:
I think I would separate this into a few tests:
1. Get changelogs when none exist
2. Get changelogs returns as expected
3. Get changelogs returns multiple (which is this existing test)

Reply:
I can add for the none test case, but I'm a bit confused between 2 and 3. So for the expected one should I just have it return a single change and test its format ?

Reply:
The difference between 2 and 3 is that test 2 checks that "this should return something" (it is the opposite of test 1) but it doesn't care **what** is returned. Test 3 checks **what is returned** (it is formatted correctly? how many items are returned? etc). You don't HAVE to have test 2, it's just a good excuse to write more tests ;)